backend/app/core/review_analyzer.py

```
@@ -280,6 +282,16 @@ async def _analyze_single_batch(self, reviews: list[dict]) -> dict | None:
                     logger.error("LLM 분석 최대 재시도 초과, 이 배치 건너뜀")
                     return None
+            except (httpx.HTTPStatusError, httpx.TimeoutException, httpx.ConnectError) as e:
+                # Cloudflare 504 / 네트워크 일시 오류 등 — 지수 백오프 후 재시도
+                wait = 2 ** attempt
+                logger.warning(
+                    f"LLM HTTP 오류 (시도 {attempt + 1}/{max_retries + 1}): {e} → {wait}초 후 재시도"
+                )
+                if attempt == max_retries:
+                    logger.error("LLM HTTP 최대 재시도 초과, 이 배치 건너뜀")
+                    return None
+                await asyncio.sleep(wait)
```

Ruff (0.15.10)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)